node/node.go

```diff
@@ -433,6 +433,12 @@ func (n *nodeImpl) OnStart(ctx context.Context) error {
 		return fmt.Errorf("cannot load state: %w", err)
 	}
 
+	if len(n.peerManager.Peers()) == 0 && !onlyValidatorIsUs(state, n.nodeKey.PubKey()) {
```

Comment:
This is fine although to be more precise I think `onlyValidatorIsUs` should be `doWeHaveMajority`

Reply:
I do not disagree with you